api/resources_portal/test/dev_data/grants.json

```diff
@@ -0,0 +1,87 @@
+"grants": {
```

Comment:
Oh, I didn't realize this was in the `test` directory. It should be moved up because it's not actually for the tests.

Reply:
Actually, I think it really belongs in `api/`, not `api/resources_portal/` because it's not part of the Django app.